kubejs/server_scripts/tfc/recipes.js

```diff
@@ -518,7 +518,7 @@ const registerTFCRecipes = (event) => {
                         
                         // Винт -> Металл
                         event.recipes.tfc.heating(screwItem, tfcProperty.getMeltTemp())
-                            .resultFluid(Fluid.of(outputMaterial.getFluid(), 72))
+                            .resultFluid(Fluid.of(outputMaterial.getFluid(), 16))
```

Comment:
Should be 18, because:
![image](https://github.com/TerraFirmaGreg-Team/Modpack-1.20.x/assets/52341158/505e894f-024f-4a76-a4c5-44f9837abd21)

Reply:
![image](https://github.com/TerraFirmaGreg-Team/Modpack-1.20.x/assets/32123043/377599de-7756-4e43-a5ef-04e4ef174b72)

If you set as 18mB, players can "grind" TFC metals, since GT extractor produce 16mB molten metals from 1 screw. (See picture)

Reply:
Funny gtceu again, ok then, 16mb.

Reply:
> Funny gtceu again, ok then, 16mb.

I think this is because screws are produced by **milling** bolts, during which some tiny metal dusts are also produced. However, since GTCEu does not have smaller dust than "tiny dusts" (which equals to 1/9 ingot), these dusts are counted as lost.